components/Premium/BuyPremium/BuyPremium.tsx

```diff
@@ -71,6 +71,9 @@ function BuyPremium() {
                                 <span>{numberWithThousandsSeperators(coflCoins - purchasePremiumDuration * PREMIUM_PRICE_MONTH)} Coins</span>
                             </div>
                         ) : null}
+                        <p style={{ marginTop: '20px' }}>
+                            This is not a recurring subscription. We won't automatically charge you after your premium time runs out!
```

Comment:
```suggestion
                            This is a prepaid service. We won't automatically charge you after your premium time runs out!
```
reduce inversions, inversions are harder for brains to understand.